api/helpers/things.js

```diff
@@ -196,6 +196,72 @@ function setConditional(
   }
 }
 
+//get keylist and keys by methods,cases, orgs
+
+const facetKeys = (type) => {
```

Comment:
@pan what do you think about replacing `facet*` with `searchFilters*` here to match the naming convention on the front-end?